Assets/MRTK/Services/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -57,9 +60,6 @@ public float GestureBlending
             set
             {
                 gestureBlending = Mathf.Clamp(value, gestureBlending, 1.0f);
-
-                // Pinch is a special gesture that triggers the Select and TriggerPress input actions
-                IsPinching = (gesture == ArticulatedHandPose.GestureId.Pinch && gestureBlending > 0.9f);
```

Comment:
It looks like you're removing the check for gesture blending will this cause the select gesture to happen immediately, rather than after a small delay? Are there any other side effects of this?

Reply:
Yeah the main purpose of this was to remove the dependency of the IsPinching flag to the gesture animation. There shouldn't be any other side effects based on the unit testing